archive/bagger/src/main.py

```diff
@@ -41,7 +41,9 @@ def process_message(message):
     print("processing " + identifier)
     result = bagger_processor.process_bagging_message(body)
     error = result.get("error", None)
-    if error is not None:
+    if error is None:
```

Comment:
I assume the running bagger still uses this `main` function as it processes the queue?

If so, if no errors during bagging, get rid of any error previously logged to the error bucket.